.github/workflows/release.yml

```diff
@@ -290,6 +272,63 @@ jobs:
           asset_name: k0s-airgap-bundle-${{ needs.release.outputs.tag_name }}-arm
           asset_content_type: application/octet-stream
 
+
+  build-image:
+    needs:
+      - release
+      - x64
+      - arm64
+      - armv7
+    runs-on: [self-hosted,linux,x64]
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Fetch k0s amd64
+        uses: actions/download-artifact@v2
+        with:
+          name: k0s-amd64
+      - name: Fetch k0s arm64
+        uses: actions/download-artifact@v2
+        with:
+          name: k0s-arm64
+      - name: Fetch k0s arm
+        uses: actions/download-artifact@v2
+        with:
+          name: k0s-arm
+
+      - name: Build image and push to Docker Hub and GitHub image registry
+        uses: docker/build-push-action@v2
+        with:
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
+          tags: |
+            ghcr.io/k0sproject/k0s:${{ needs.release.outputs.image_tag }}
+            docker.io/k0sproject/k0s:${{ needs.release.outputs.image_tag }}
+          push: true
+
+      # Need to remove this from maintenance branches
+      # Done as separate step for better control when we push latest
+      - name: Build image and push to Docker hub
+        if: "!contains(github.ref, '-')"
+        uses: docker/build-push-action@v2
+        with:
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
+          tags: docker.io/k0sproject/k0s:latest
+          push: true
+
```

Comment:
I'm thinking if we should just stop pushing `latest` in general. It's a bad advise in general to use latest tags and for us it's bit problematic to manage which ones are actually latest tags.

Reply:
Yeah, it could be useful only for some testing or development purposes. Should I remove it or do we need some discussion?

Reply:
I'd say we can safely drop it.

Reply:
Deleted it